assets/js/workflow-diagram/WorkflowDiagram.tsx

```diff
@@ -590,15 +590,18 @@ export default function WorkflowDiagram(props: WorkflowDiagramProps) {
         liveAction={props.liveAction}
         drawerWidth={drawerWidth}
       />
-      <MiniHistory
-        collapsed={!runSteps.start_from}
-        history={someHistory}
-        selectRunHandler={onRunChange}
-        onCollapseHistory={onCollapseHistory}
-        drawerWidth={drawerWidth}
-        hasSnapshotMismatch={hasSnapshotMismatch}
-        missingNodeCount={missingNodeCount}
-      />
+      {
+        props.liveAction === 'edit' ?
```

Comment:
Just for curiosity sake and learning JS, is this better than `props.liveAction === 'edit' && ...` ?